.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+on:
+  push:
+    branches: [main]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - uses: olafurpg/setup-scala@v10
+      - run: sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
```

Comment:
Hmmm actually... are all these 4 secrets some VirtusLab-wide creds? how have you gotten them, is there some LastPass that I can have access to as well?

Reply:
I got these secrets form forwarded emails

Reply:
![image](https://user-images.githubusercontent.com/3383210/132006567-e2bf6eec-a4f6-420e-8a98-e23f2e94274c.png)